squidDrone/.clang-format

```diff
@@ -1,4 +0,0 @@
-Language: Cpp
-BasedOnStyle: Google
```

Comment:
I'm missing this file.

Reply:
Readded in [this commit](https://github.com/elheck/Squiddrone/pull/239/commits/6bf0a177d6d45301e222f00792db99885480eedb)